app/src/main/java/edu/calvin/cs262/teamc/MyMatchesActivity.java

```diff
@@ -51,35 +59,34 @@ protected void onCreate(Bundle savedInstanceState) {
         lv = ((ListView) findViewById(R.id.myMatchesLv));
 
 
-        String requestUrl = String.format("https://calvincs262-fall2018-teamc.appspot.com/pinder/v1/matches/%d", personID);
-        Log.d("MyMatches", requestUrl);
+        String requestUrl = String.format(
+                "https://calvincs262-fall2018-teamc.appspot.com/pinder/v1/matches/%d", personID);
+
+        StringRequest stringRequest = new StringRequest(Request.Method.GET,
+                requestUrl, new Response.Listener<String>() {
 
-        StringRequest stringRequest = new StringRequest(Request.Method.GET, requestUrl, new Response.Listener<String>() {
             @Override
             public void onResponse(String response) {
-             //   Toast.makeText(MyMatchesActivity.this, "Recieved Information!", Toast.LENGTH_SHORT).show();
                 try {
+                    // Convert response to JSON
                     JSONObject jsonObj = new JSONObject(response);
                     JSONArray arrJson = jsonObj.getJSONArray("items");
-                    //Log.e("JSON object",arrJson.toString());
+                    // Add each match to memory
                     for (int i=0; i<arrJson.length(); i++)
                     {
                         JSONObject object = arrJson.getJSONObject(i);
-                        Log.e("GetDogs-Name",object.getString("dogName"));
                         String Name =object.getString("dogName");
-                        Log.e("GetDogs-Breed",object.getString("dogBreed"));
                         String Breed =object.getString("dogBreed");
-                        Log.e("GetDogs-ProfilePhoto",object.getString("profilePicture"));
                         String Photo =object.getString("profilePicture");
 
                         MatchInfo mi = new MatchInfo(Name,Breed,Photo);
                         matches.add(mi);
 
```

Comment:
These few empty lines could maybe be deleted too.

Reply:
Deleted